Content.Client/Power/APC/ApcVisualizer.cs

```diff
@@ -15,69 +10,15 @@ public sealed class ApcVisualizer : AppearanceVisualizer
         public static readonly Color FullColor = Color.FromHex("#3db83b");
         public static readonly Color EmagColor = Color.FromHex("#1f48d6");
 
-        [UsedImplicitly]
-        [Obsolete("Subscribe to your component being initialised instead.")]
-        public override void InitializeEntity(EntityUid entity)
-        {
-            base.InitializeEntity(entity);
-
-            var sprite = IoCManager.Resolve<IEntityManager>().GetComponent<SpriteComponent>(entity);
-
-            sprite.LayerMapSet(Layers.Panel, sprite.AddLayerState("apc0"));
-
-            sprite.LayerMapSet(Layers.ChargeState, sprite.AddLayerState("apco3-0"));
-            sprite.LayerSetShader(Layers.ChargeState, "unshaded");
-
-            sprite.LayerMapSet(Layers.Lock, sprite.AddLayerState("apcox-0"));
-            sprite.LayerSetShader(Layers.Lock, "unshaded");
-
-            sprite.LayerMapSet(Layers.Equipment, sprite.AddLayerState("apco0-3"));
-            sprite.LayerSetShader(Layers.Equipment, "unshaded");
-
-            sprite.LayerMapSet(Layers.Lighting, sprite.AddLayerState("apco1-3"));
-            sprite.LayerSetShader(Layers.Lighting, "unshaded");
-
-            sprite.LayerMapSet(Layers.Environment, sprite.AddLayerState("apco2-3"));
-            sprite.LayerSetShader(Layers.Environment, "unshaded");
-        }
-
         [Obsolete("Subscribe to AppearanceChangeEvent instead.")]
```

Comment:
you should convert this into a system instead of keeping it as a visualizer, this is obsolete

Reply:
ik, i just do it gradually

Reply:
since you’re already trimming it down and it’s a pretty small visualizer, it wouldn’t hurt to just do it now

Reply:
I will try

Reply:
Thought sloth already approged this so I will make it in next round